packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -52,8 +52,7 @@ const PipelineRuns = ({
   },
   getPipelineRunCreatedTime = pipelineRun =>
     pipelineRun.metadata.creationTimestamp,
-  getPipelineRunId = pipelineRun =>
-    `${pipelineRun.metadata.namespace}:${pipelineRun.metadata.name}`,
+  getPipelineRunId = pipelineRun => pipelineRun.metadata.uid,
```

Comment:
@steveodonovan I'm not sure why we were using namespace+name before, makes more sense to me to use the uid and not seeing any other impact of this change other than it being easier to resolve selected rows to actual resources. We're already using uid for TaskRuns.